### DISCUSSION TITLE
public app - ping "distinct ID" error

### DISCUSSION THREAD

**Discussion by gab-elio** [Budibase/budibase#5336] [Help]
Hi, 
I'm building a public landing page fetching data from an external mongo db. preview works well, but when i  try a public access i have got an error "you must pass a distinct id" 
![image](https://user-images.githubusercontent.com/102796798/162585162-010b15a7-afb7-492e-ba79-6ac209a7832c.png)

This error is due to the ping request sent to budibase (I work on the cloud version)
![image](https://user-images.githubusercontent.com/102796798/162585185-74fe7d67-6f55-4687-bf2c-9f8eb8db2f37.png)
![image](https://user-images.githubusercontent.com/102796798/162585223-780346bd-9589-4c52-a586-1845f36c0fbd.png)

Do you have an idea on how to pass that distinct id, or disable theses requests (or hide error notification on published environnement)   ?

Many thanks.

**Comment by melohagan** [comment]
Hi @gab-elio 

I think this may simply be a session issue. If you haven't tried already, could you log out and log in again please?

**Reply by gab-elio** [reply]
Hi @melohagan 
this issue is when I use the app as a anonymous user, so there is no way to log in/out.  When logged in, I don't have that problem. 
(still tried to log in / out; didn't change any thing)

**Comment by melohagan** [comment]
@gab-elio 
This is a bug, which I have raised here: https://github.com/Budibase/budibase/issues/5351

**Reply by gab-elio** [reply]
thanks :)
